xml/bp_troubleshooting_tips.xml

```diff
@@ -318,23 +318,21 @@ osd_scrub_load_threshold = 0.25
 
   <tip>
    <para>
-    Create the <filename>/var/lib/ceph</filename> subvolume before running
-    &deepsea; stage 0 because stage 0 installs &ceph; related packages and
-    creates the <filename>/var/lib/ceph</filename> directory.
+    Create the <filename>/var/lib/ceph</filename> subvolume before installing
```

Comment:
This whole section is IMO a good candidate for removal as MONs (and all services) run in containers in SES7

Reply:
Okay, I'll remove this section and add a note for future Alex.